sdcm/argus_results.py

```diff
@@ -365,12 +377,22 @@ def set_validation_rules(column_metadata):
     submit_results_to_argus(argus_client, result_table)
 
 
-def send_manager_benchmark_results_to_argus(argus_client: ArgusClient, result: dict, sut_timestamp: int,
-                                            row_name: str = None) -> None:
+def send_manager_benchmark_results_to_argus(argus_client: ArgusClient,
+                                            result: dict,
+                                            sut_timestamp: int = 0,
+                                            row_name: str = None,
+                                            report_type: Literal["regular", "one_to_one"] = "regular") -> None:
     if not row_name:
         row_name = "#1"
 
-    result_table = ManagerRestoreBanchmarkResult(sut_timestamp=sut_timestamp)
+    match report_type:
```

Comment:
if the results differ only by `ManagerOneOneRestoreBenchmarkResult` why not make the result_table as a parameter instead?